src/components/select/select.tsx

```diff
@@ -281,7 +281,7 @@ export class Select {
         const element: HTMLSelectElement = this.host.shadowRoot.querySelector(
             'select.limel-select__native-control'
         );
-        const options = Array.apply(null, element.options)
+        const options = Array.apply(null, element.options) // eslint-disable-line prefer-spread
```

Comment:
I assume all this is doing is making a copy of the array so we don't mutate the original one? Wouldn't the spread operator do the same thing here?

Reply:
I don't think the spread operator will work here because `element.options` is a `DOMCollection` or something similar